apps/src/code-studio/components/header/ScriptName.jsx

```diff
@@ -126,13 +133,18 @@ const styles = {
   },
   headerInner: {
     position: 'absolute',
+    display: 'flex',
+    alignItems: 'center',
   },
   scriptLinkWithUpdatedAt: {
     display: 'block',
   },
   outerContainer: {
     textAlign: 'right',
```

Comment:
nit: have you tried using `textAlign: 'end'` here? This will let css to automatically align text to right in ltr and left in rtl language and you won't need to add `outerContainerRtl` at all

Reply:
Nice, updated.